awx/main/migrations/0018_add_generic_command_event.py

```diff
@@ -0,0 +1,39 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.7 on 2017-12-13 23:00
+from __future__ import unicode_literals
+
+import awx.main.fields
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('main', '0017_v330_move_deprecated_stdout'),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='GenericCommandEvent',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('created', models.DateTimeField(default=None, editable=False)),
+                ('modified', models.DateTimeField(default=None, editable=False)),
+                ('event', models.CharField(choices=[(b'runner_on_failed', 'Host Failed'), (b'runner_on_ok', 'Host OK'), (b'runner_on_unreachable', 'Host Unreachable'), (b'runner_on_skipped', 'Host Skipped'), (b'debug', 'Debug'), (b'verbose', 'Verbose'), (b'deprecated', 'Deprecated'), (b'warning', 'Warning'), (b'system_warning', 'System Warning'), (b'error', 'Error')], max_length=100)),
+                ('event_data', awx.main.fields.JSONField(blank=True, default={})),
+                ('failed', models.BooleanField(default=False, editable=False)),
+                ('changed', models.BooleanField(default=False, editable=False)),
+                ('uuid', models.CharField(default=b'', editable=False, max_length=1024)),
+                ('counter', models.PositiveIntegerField(default=0, editable=False)),
+                ('stdout', models.TextField(default=b'', editable=False)),
+                ('verbosity', models.PositiveIntegerField(default=0, editable=False)),
+                ('start_line', models.PositiveIntegerField(default=0, editable=False)),
+                ('end_line', models.PositiveIntegerField(default=0, editable=False)),
+                ('unified_job', models.ForeignKey(editable=False, on_delete=django.db.models.deletion.CASCADE, related_name='generic_command_events', to='main.UnifiedJob'))
+            ],
+            options={
+                'abstract': False,
```

Comment:
Why does it think this model is abstract? This is the concrete one. Do we need the `Meta` class on it?

Reply:
I'm not sure why Django put this in here, but notice that it's `abstract=False`, which _is_ accurate.